ztp/ztp-ran-policy-generator/sourcePolicies/PerformanceProfile.yaml

```diff
@@ -3,8 +3,7 @@ kind: PerformanceProfile
 metadata:
   name: $name
 spec:
-  additionalKernelArgs:
-    - nosmt
+  additionalKernelArgs: []
```

Comment:
if it is an empty array, it could be better to remove it.

Reply:
Definitely could remove it, but having the empty array is a nice placeholder to let the user know what can be updated if they want.